src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ CHANGELOG
  * Add support for per-env configuration in XML and Yaml loaders
  * Add `ContainerBuilder::willBeAvailable()` to help with conditional configuration
  * Add support an integer return value for default_index_method
+ * Add `#[When(env: 'foo')]` to skip autoregistering a class when the env doesn't match
```

Comment:
Is it okay for you if we write this sentence in a positive way?
```suggestion
 * Add `#[When(env: 'foo')]` to autoregister a class when the env matches
```

Reply:
Although, we will use this attribute only when we really need skipping autoregistration, maybe the negative one is better.

Reply:
I think the negative one is more clear yes.